code/modules/mob/living/carbon/human/death.dm

```diff
@@ -101,13 +101,12 @@
 	SetJitter(0)
 	heart_attack = 0
 
+	for(var/obj/item/organ/O in organs) //these are external organs only: arms, legs, etc, internal organs are handled by carbon/death()

```

Comment:
organs is a list with the external organs only, where is it looping through internal organs the second time?